app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/NoConnectionSimple.kt

```diff
@@ -0,0 +1,97 @@
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.aptoide.android.aptoidegames.theme.agWhite
+import com.aptoide.android.aptoidegames.theme.blueGradientEnd
+import com.aptoide.android.aptoidegames.theme.blueGradientStart
+
+@Preview
+@Composable
+fun TestNoConnectionSimple() {
+  Image(
+    imageVector = getNoConnectionSimple(),
+    contentDescription = null,
+    modifier = Modifier.size(64.dp)
+  )
+}
+
+fun getNoConnectionSimple(): ImageVector = ImageVector.Builder(
+  name = "NoConnectionSimple",
+  defaultWidth = 64.dp,
+  defaultHeight = 64.dp,
+  viewportWidth = 64f,
+  viewportHeight = 64f,
+).apply {
+  path(
+    fill = Brush.linearGradient(
+      colors = listOf(blueGradientStart, blueGradientEnd),
+      start = Offset(x = 5.444f, y = 6.326f),
+      end = Offset(x = 69.915f, y = 38.658f)
+    ),
+  ) {
+    moveTo(32f, 64f)
+    curveTo(49.673f, 64f, 64f, 49.673f, 64f, 32f)
+    curveTo(64f, 14.327f, 49.673f, 0f, 32f, 0f)
+    curveTo(14.327f, 0f, 0f, 14.327f, 0f, 32f)
+    curveTo(0f, 49.673f, 14.327f, 64f, 32f, 64f)
+    close()
+  }
+  path(
+    fill = SolidColor(agWhite),
```

Comment:
Let's not hardcode here but provide it through arguments.
Better set one of the new colours in the place it is used.